src/collections/_documentation/workflow/alerts-notifications/notifications.md

```diff
@@ -0,0 +1,87 @@
+
+For example:
+
+- You're subscribed to the issue or all workflow emails in the project
+- You're involved in a commit related to the issue
+- You commented on or bookmarked the issue
+- You / your team was mentioned in the issue or assigned the issue
+
+### Issue States
+
+An issue can be in one of the following three states:
+
+**Unresolved**
+
+- The default state when an issue is added to the system.
+
+**Resolved**
+
+- An issue is marked as resolved when an individual resolves it by manually changing the state on the issue page UI, submitting a fix with the issue ID in it, or when the project’s auto-resolve feature is configured. Auto-resolve is located in **Project Settings > General Settings**. For more details about submitting a fix, see documentation on [resolving issues](https://docs.sentry.io/workflow/releases/#after-associating-commits).
+
+    ![https://docs.sentry.io/assets/notifications/auto-resolve-4ff17e17a825d4e2f16293d1e3371666838dcf20630c2caa61fc3af609289db3.png](https://docs.sentry.io/assets/notifications/auto-resolve-4ff17e17a825d4e2f16293d1e3371666838dcf20630c2caa61fc3af609289db3.png)
+
+- Resolve an issue to indicate that no more events are expected. If another event is seen in the issue, the issue will revert to unresolved. This is called a **regression**.
+
+**Ignored**
+
+- Ignored issues suppress alerts for the issue, and by default, are hidden from your project’s issue stream. Search the issue stream for `is:ignored` to view them.
+
+Workflow emails are sent for issue state changes.
+
+### Other workflow notifications
+
+- When you or your team is **Assigned or Unassigned** an issue
+- When someone leaves a comment
+
+## Deploy
+
+Deploy emails are sent to users who have committed to the release that was deployed. See [LINK: deploy documentation] for more details
+
+[ screenshot deploy-emails.png ]
+
+## Quota
+
+Sentry sends all owners of the organization emails when
+
+- The event volume is approaching or has exceeded the quota
+- On-demand is activated
+- Spike protection is activated or deactivated
+
+## **Notification Subscription**
+
+You can configure your personal workflow and deploy notifications [here](https://sentry.io/settings/account/notifications/).
+
+### **Workflow Notifications**
+
+![https://docs.sentry.io/assets/notifications/workflow_notifications_subscription-1f12011dbcd124436fb79f20b858eb338789fc530e4fa561f0535101a9d57e82.png](https://docs.sentry.io/assets/notifications/workflow_notifications_subscription-1f12011dbcd124436fb79f20b858eb338789fc530e4fa561f0535101a9d57e82.png)
+
+You can also click “Fine-tune workflow notifications by project” to configure this with more granularity. Selecting Default will use your preference from the last step for that project.
```

Comment:
should we repeat this with the same level of detail as alert preferences? i.e. 3 options, what default means etc.